lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -162,9 +162,10 @@ class HttpApiEvents {
         DependsOn: this.provider.naming.getHttpApiIntegrationLogicalId(targetData.functionName),
       });
       if (authorizer) {
+        const { authorizerId } = authorizer;
```

Comment:
This doesn't follow proposal from the issue, as put here: https://github.com/serverless/serverless/issues/7598#issuecomment-623300695

If you do not agree with it, and feel that you have better idea, please comment and propose in an issue.

After we agree on final implementation, let's focus on PR

Reply:
I've updated the PR based on agreement in the issue ;)